### DISCUSSION TITLE
Default Applications and Wi-Fi Issues on Arch Linux

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1549] [General]
I've been having some issues on my Arch Linux system and could really use some help.

  1-  File Associations: Every time I open a file (photo, video, etc.), I have to manually select an application. This started happening again even after I previously fixed it by installing kservice5.

  2-  Wi-Fi Stability: Since the recent update, my Wi-Fi (MediaTek MT7921) keeps disconnecting every few minutes and asks for the password again.

I've tried updating everything and tweaking some settings, but nothing seems to work.

**Comment by kRHYME7** [comment]
1) 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8c9ef080-c5d8-4e4c-9fee-f218da853179)


2) check the wiki
*  https://archlinux.org/
* https://www.google.com/search?q=Arch+linux++MediaTek+MT7921&client=firefox-b-d&sca_esv=2b3402ed1f730118&sxsrf=ADLYWIIEjCiuGdq9fU0ZXB7sv8cncGLAJw%3A1716789366519&ei=diBUZrasH4jvseMP_tGV-Ak&ved=0ahUKEwi2v-Hikq2GAxWId2wGHf5oBZ8Q4dUDCBA&uact=5&oq=Arch+linux++MediaTek+MT7921&gs_lp=Egxnd3Mtd2l6LXNlcnAiG0FyY2ggbGludXggIE1lZGlhVGVrIE1UNzkyMTIIEAAYgAQYogQyCBAAGIAEGKIEMggQABiABBiiBEiAFFC7BljoEnACeACQAQGYAecBoAHKDaoBBjAuMTEuMbgBA8gBAPgBAZgCB6AC4QXCAgoQABiwAxjWBBhHwgIGEAAYBxgewgIIEAAYBxgIGB7CAgsQABiABBiGAxiKBZgDAIgGAZAGCJIHAzIuNaAHzB0&sclient=gws-wiz-serp
* Try to downgrade your kernel
* Try to use more friendly distro like Endeavor OS

Note that if this is after an upgrade, you probably updated your kernel and it broke your wifi driver. You can wait for the fix or downgrade.  

> asks for the password again. 
But if this happens frequently,   try ` service network-manager restart `

**Reply by vyrx-dev** [reply]
i got a  solution (https://www.reddit.com/r/kde/comments/1bd313p/comment/kygab1d/?onetap_auto=true&one_tap=true) on this site
